### ISSUE TITLE
Hook support removed from voyager 1.5*

### ISSUE THREAD

**Issue by rohankhudedev** [larapack/hook-docs#6]
How can we now use these hooks in voyager 1.5*?

**Comment by fletch3555** [comment]
Correct, Hooks have been removed from Voyager 1.5.  There were multiple issues with them, and the Voyager team does not have adequate access to the larapack systems to resolve them.
